electron/helpers/utils.js

```diff
@@ -44,7 +44,7 @@ const harmfulSwitches = [
 ]
 
 function isHarmfulSwitchesPresent () {
-  for (harmfulSwitch of harmfulSwitches) {
+  for (const harmfulSwitch of harmfulSwitches) {
```

Comment:
constants are not meant to be here. constants are stored in the .data section of the asm execution file in memory. if you could freeze the memory and extract out the data, you can change the whole codebase. this makes the app more vulnerable.

Reply:
```suggestion
  for (harmfulSwitch of harmfulSwitches) {
```

Reply:
@xyloflake Then it says `harmfulSwitch` is not defined.
It can be `let`, I guess

Reply:
> @xyloflake Then it says `harmfulSwitch` is not defined. It can be `let`, I guess

yes it can be `let`

```suggestion
  for (let harmfulSwitch of harmfulSwitches) {
```